profiles/kentik_snmp/aruba/aruba-clearpass.yml

```diff
@@ -10,8 +10,6 @@ provider: kentik-clearpass
 sysobjectid:
   - 1.3.6.1.4.1.14823.1.6.1    # C2000V
 
-provider: kentik-clearpass
```

Comment:
I think we dont want to completely remove this line, just remove the extra hyphen and white space

Reply:
There's another copy of the line right above.